.github/label_to_label.yml

```diff
@@ -48,3 +48,12 @@
   - "module: dynamic shapes"
   then:
   - "oncall: pt2"
+- any:
+  - "release notes: distributed (c10d)"
+  - "release notes: distributed (symm_mem)"
+  - "release notes: distributed (pipeline)"
+  - "release notes: distributed (fsdp)"
+  - "release notes: distributed (dtensor)"
+  - "oncall: distributed"
+  then:
+  - "ciflow/h100-distributed"
```

Comment:
cc @clee2000 if you have any thoughts here, maybe this is only meant to be used on issue

Reply:
This file is only used for issues

Reply:
hmmm, if this is the case what is the recommended way for PR? @clee2000 @huydhn

Reply:
I think it's referred to the logic here https://github.com/pytorch/test-infra/blob/main/torchci/lib/bot/autoLabelBot.ts#L149-L155